english/mmacoder/mmacoder.md

```diff
@@ -0,0 +1 @@
+[mmacoder.com](http://www.mmacoder.com) is where I will be blogging about my experience with the Udacity Nanodegree Plus program. Follow me on my blog and everywhere on social media to witness the rise of MMACoder to MMA and mobile app development stardom!
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.